### DISCUSSION TITLE
Support for pnfs in the NFS csi driver

### DISCUSSION THREAD

**Discussion by SchoolGuy** [kubernetes-csi/csi-driver-nfs#739] [General]
Hey guys!

A quick Google and search through the issues and discussions didn't give me an answer but is it possible to utilize pNFS with the 4.8 of the NFS CSI?

Thanks for your answers in advance!

**Comment by andyzhangx** [comment]
this driver is not bound to any nfs version, it leverages the nfs kernel driver on the node, so if you agent node supports pNFS, then it should work. @SchoolGuy
